modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc

```diff
@@ -6,7 +6,7 @@
 [id="lvms-unstalling-lvms-with-web-console_{context}"]
 = Uninstalling {lvms} installed using the OpenShift Web Console
 
-You can unstall {lvms} using the Red Hat OpenShift Container Platform Web Console.
+You can uninstall {lvms} using the Red Hat OpenShift Container Platform Web Console.
```

Comment:
Please add a {nbsp} between Red and Hat. This prevents the company name from breaking across lines in the docs.

Reply:
Also, I will add that we do not capitalize web console.
```suggestion
You can uninstall {lvms} using the Red Hat OpenShift Container Platform web console.
```
Also should be lower case in the module title as well.

Reply:
I also would like to add that you might want to consider revising some of the prerequisites and steps to this procedure to adhere more to the style guide.  To list a few for example:

- We should avoid directional language such as "right-hand side" used on lines 23 and 25. 
- s/drop-down menu/list
- "You have access to the cluster using an account with `cluster-admin` permissions." should be first prereq because it is essential to do the rest of the prereqs and the procedure.
- Step 1, consider revising to remove "scroll to *LVM Storage* or type `LVM Storage` into the *Filter by name* to find and click on it." you can just say "click *LVM Storage*"
- Step 2,3, and 4 could be one step: "Click the *LVMCluster* tab and select *Delete LVMCluster* and *Uninstall Operator* from the *Actions* list.
- Last step: "Click *Remove*"

I know it is out of scope of this PR, so you could do it here or create a new issue for it. Feel free to ping me if you have questions. 

Ps this confluence page is a good resource to help with UI elements: https://docs.engineering.redhat.com/pages/resumedraft.action?draftId=380770273&draftShareId=71b5b07a-7293-46dd-ae22-f71ebbc24066&

Reply:
@mjpytlak 
Thank you for the review.
I've removed "Red Hat" as it is not used in other modules of the assembly.
To maintain consistency, the line and the title have been updated.
PTAL.